content/js/query.md

```diff
@@ -183,12 +191,22 @@ multiple keys, but doing so will impair performance. If you don't know about
 reference, please read [Records](/js/guide/record#reference) section first.
 
 ``` javascript
-let query = new skygear.Query(Note);
-query.transientInclude('nextPage', '<your-transient-name>');
+// Suppose we have Delivery and Address
+const Delivery = skygear.Record.extend('delivery');
+const Address = skygear.Record.extend('address');
+
+// Suppose delivery has set destination reference to address
+var address = new Address({ ... });
+var delivery = new Delivery({ destination: skygear.Reference(address) });
+skygear.publicDB.save([address, delivery]).then(...);
+
+// Now when we are retrieving delivery, we want to include address as well
+var query = new skygear.Query(Delivery);
+query.transientInclude('destination', 'delivery-address');
```

Comment:
need to mention:

- the first argument is the column of the delivery record
- the second argument is to specify the key that the transient record should be included in `record.$transient`
- the second argument is optional, default to the column name